addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -1,22 +1,15 @@
-/* global YT */
-var onYouTubeIframeAPIReady = undefined;
```

Comment:
I think this is broken, the page really doesn't render what it should

I see `<iframe src="https://9592176-master-all.runbot32.odoo.com/slides/embed/1?page=1" class="o_wslides_iframe_viewer" allowFullScreen="true" height="315" width="420" frameborder="0"></iframe>`  as text?

I don't want to delay the PR as runbot is green and rebase is nightmare, but this should be looked into first thing tomorrow.

Reply:
I will check now